### PR TITLE
Creation of dog commmand

### LINKED ISSUE #3: Dog Pictures

### PR DESCRIPTION
**Pull request by LiamTownsley**
Resolves #3 
image

### CODE REVIEW

File: src/commands/dog.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const fetch = require('node-fetch');
```
Comment (groveld) [inline src/commands/dog.js:1]: please add node-fetch to package.json

**Review by groveld — CHANGES_REQUESTED** [review]
Please add the NPM package for node-fetch.

File: src/commands/dog.js (hunk 1 above)
Comment (LiamTownsley) [reply src/commands/dog.js:1]: yeah sorry about that, i already had it installed on my bot, one second

**Review by groveld — APPROVED** [review]
Looks good.

**Pull request merged**